backend/src/test/java/de/bund/digitalservice/ris/norms/adapter/input/scheduler/SchedulerTest.java

```diff
@@ -1,38 +0,0 @@
-package de.bund.digitalservice.ris.norms.adapter.input.scheduler;
-
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-
-import de.bund.digitalservice.ris.norms.application.port.input.PublishNormUseCase;
-import java.util.concurrent.TimeUnit;
-import org.awaitility.Awaitility;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
-import org.springframework.boot.test.mock.mockito.MockBean;
-import org.springframework.boot.test.mock.mockito.SpyBean;
-import org.springframework.scheduling.annotation.EnableScheduling;
-import org.springframework.test.context.TestPropertySource;
-import org.springframework.test.context.junit.jupiter.SpringExtension;
-
-@ExtendWith(SpringExtension.class)
-@EnableScheduling
-@TestPropertySource(properties = "publish.cron=*/1 * * * * *") // Runs every 5 seconds
-class SchedulerTest {
```

Comment:
We dont need this test anymore? It was testing the setting of the properties and check if the job is started at all

Reply:
The test was breaking due to the AssertLock. I had the impression this test is mostly testing if the scheduler works. Under the assumption: "Don’t Test The Framework" I threw it out.

Reply:
Fine :)